server/lib/tuist_web/live/build_run_live.ex

```diff
@@ -845,9 +891,36 @@ defmodule TuistWeb.BuildRunLive do
 
     {tasks, tasks_meta} = Runs.list_cacheable_tasks(options)
 
+    # Fetch CAS outputs for all tasks on the current page
```

Comment:
this can probably be improved, the issue is we have a clickhouse many-to-many relationship where each cacheable task can have multiple cas outputs, but also each cas output can belong to multiple cacheable tasks. This works, but maybe we can model it nicer. @cschmatzler any ideas?